spec/index.bs

```diff
@@ -4782,18 +4800,28 @@ command encoder can no longer be used.
                 descriptor:
             </pre>
 
-            **Returns:** {{undefined}}
-
-            Issue the following steps on the [=Device timeline=] of |this|:
-            <div class=device-timeline>
-                1. If any of the following conditions are unsatisfied, generate a validation
-                    error and stop.
-                    <div class=validusage>
-                        - |this| is [=valid=].
-                        - |this|.{{GPUCommandEncoder/[[debug_group_stack]]}}.length is 0.
+            **Returns:** {{GPUCommandBuffer}}
+
+            Issue the following steps on the [=Content timeline=] of |this|:
```

Comment:
My assumption has been that steps run on the content timeline by default (it's the JS timeline). So wouldn't need to wrap these steps.